stix_shifter_modules/guardium/stix_transmission/api_client.py

```diff
@@ -102,12 +102,10 @@ def get_search_results(self, search_id,  index_from=None, fetch_size=None):
             indx = int(index_from)+1
             fsize = int(fetch_size)+1
             if "reportName" in self.query:
-                response = self.client.handle_report(self.query["reportName"], self.query["reportParameter"], indx, fsize)
-                respObj = ResponseWrapper(response)
+                response = self.client.handle_report(self.query, indx, fsize)
```

Comment:
why the `reportName` or `reportParameter` not used to make the api call anymore? If the they are not applicable anymore then we should also update the query construction.

Reply:
reportName and reportParameter are put into the query in the query construction step, I removed redundant code that was reassembling the query again before execution